sample_projects/LEQuad/main_avr32.cpp

```diff
@@ -72,7 +71,7 @@ int main(void)
     // Create board
     // -------------------------------------------------------------------------
     megafly_rev4_conf_t board_config    = megafly_rev4_default_config();
-    board_config.imu_config             = imu_config();                         // Load custom imu config (cf conf_imu.h)
```

Comment:
uncommented in https://github.com/lis-epfl/MAVRIC_Library/pull/311/commits/674fc5fd12fdc1576ff7ea165e6324663a3e1443